Source/ProjectNL/Ai/BTDecorator/BTDecorator_CheckGamePlayTag.cpp

```diff
@@ -0,0 +1,65 @@
+// Fill out your copyright notice in the Description page of Project Settings.
+
+
+#include "ProjectNL/Ai/BTDecorator/BTDecorator_CheckGamePlayTag.h"
+#include "AIController.h"
+#include "BehaviorTree/BlackboardComponent.h"
+#include "AbilitySystemComponent.h"
+
+UBTDecorator_CheckGamePlayTag::UBTDecorator_CheckGamePlayTag()
+{
+	NodeName = "Check Gameplay Tags"; // BehaviorTree에서 보여질 이름
+}
+
+bool UBTDecorator_CheckGamePlayTag::CalculateRawConditionValue(UBehaviorTreeComponent& OwnerComp, uint8* NodeMemory) const
+{
+	AAIController* OwnerController = Cast<AAIController>(OwnerComp.GetOwner());
+	APawn* ControlledPawn = OwnerController ? OwnerController->GetPawn() : nullptr;
+
+	// PerformConditionCheckAI를 호출해 조건 검사
+	return PerformConditionCheckAI(OwnerController, ControlledPawn);
+}
+
+bool UBTDecorator_CheckGamePlayTag::PerformConditionCheckAI(AAIController* OwnerController, APawn* ControlledPawn) const
+{
+	if (!OwnerController || !ControlledPawn)
+	{
+		return false;
+	}
```

Comment:
```suggestion
	if (!OwnerController)
	{
		return false;
	}
	
	if (!ControlledPawn)
	{
	       return false;
	}
```
저는 이게 더 직관적일 수 있어보이는데 어떻게 생각하세요? (제 개인 취향이기도 하고 이게 나중가면 가독성이 높아지는 것 같다고 생각들어서오...!)